libs/input/src/lib/phone-number/phone-number.component.ts

```diff
@@ -84,4 +84,8 @@ export class PhoneNumberComponent implements OnInit, ControlValueAccessor {
     return util.formatNumber(value, this.selectedCountry.countryCode);
   }
 
+  trackByFn(index) {
```

Comment:
This is not sufficient. To implement this, return a primitive you want to track the loop by. In this case, it could be the country code (since that will be unique).